mod/wikirate_source/template/type/source/tabs/source_details.haml

```diff
@@ -0,0 +1,10 @@
+.col-md-12.column
+  .row.clearfix
+    = nest(card, view: :listing)
```

Comment:
for nests of the same card you can call render:
`_render :listing` (underscore for skipping permission checks)